pom.xml

```diff
@@ -70,7 +70,7 @@ Note: The maven build currently needs to be started from the javaslang root dir
         <java.version>1.8</java.version>
         <jmh.version>1.13</jmh.version>
         <junit.version>4.12</junit.version>
-        <gwt.version>2.8.0-rc1</gwt.version>
+        <gwt.version>2.8.0-535f2fa9</gwt.version>
```

Comment:
This artifact must be build manually (see [README.md](https://github.com/ruslansennov/javaslang/blob/57db46ef641e22bd9168792e8a961f68bda957ff/javaslang-gwt/README.md))